.github/workflows/test.yaml

```diff
@@ -549,7 +549,8 @@ jobs:
         esac
     - name: Set up test user
       run: |
-        useradd test-user
+        useradd -U -m test-user
+        grep -n '' /etc/group /etc/passwd
```

Comment:
what is happening here?

Reply:
forgot to remove that :) anyways, it's listing contents of home and passwd